### ISSUE TITLE
questions regarding the implementation of the training code

### ISSUE THREAD

**Issue by FBehrad** [rw-ocrl/ftdinosaur-inference#4]
Dear authors,

I am implementing the training code.
I have a few questions:
1. Do you apply the cosine learning rate scheduler with warm-up only to parameters of slot attention (not the DinoV2 encoder's params)? 
2. Regarding the Blockwise LR for the encoder: according to Table C.5, LR layer i = LR decay factor * LR layer i+1. I assume this is only for the encoder part of Dinov2. What do you do for the embeddings and layer norm of Dinov2? 
3. Can you please share your implementation for P-ARI and PQ metrics? I have implemented the P-ARI, but unlike other metrics (FG-ARI and mBO), I couldn't reproduce your results, and I get around 3 percent less than what is reported in the paper. Note: I use these pretrained weights: _dinosaur_base_patch14_518_topk3.coco_dv2_ft_s7_300k+10k_, and these are my results averaged over 5 runs:
mBO_i     36.395441
mBO_c    41.274785
FG-ARI    49.133659
miou       34.965829
P-ARI     46.336267
4. In Section E (COCO), you mention "we evaluate square center crops where images are resized to 224 x 224," but in Table C.5, you have used the cropping strategy of Random without any augmentation. Does Table C.5 explain your fine-tuning setting, while the other is just for the inference? 
5. What are the effects of resizing the masks to 320 x 320? 

Thank you so much.
